include/boost/random/detail/traits.hpp

```diff
@@ -0,0 +1,107 @@
+/* boost random/detail/seed.hpp header file
```

Comment:
Should be traits.hpp.  Also, is this a public header or not?  If it's not public, it should not be included in the documentation and should not be visible to users.  If it is public, it shouldn't be under detail/.